docs/operation-log.md

```diff
@@ -17,6 +17,10 @@ The operation log allows you to undo an operation (`jj [op] undo`), which doesn'
 need to be the most recent one. It also lets you restore the entire repo to the
 way it looked at an earlier point (`jj op restore`).
 
+When referring to operations, you can use `@` to represent the current operation
+as well as `+` and `-` operators to get the descendant or an ancestor of an
```

Comment:
`-` gets the parent operations, not all ancestors

Reply:
Also a good point.